cmake/Modules/FindMKLDNN.cmake

```diff
@@ -43,15 +43,15 @@ IF(NOT MKLDNN_FOUND)
       endif()
     endif()
     if(LINUX)
-      set(ABI_NEUTRAL_FLAGS -fpreview-breaking-changes)
+      set(DNNL_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fpreview-breaking-changes")
```

Comment:
@min-jean-cho , why did you remove the `else` branch? Current block is Linux only!

Reply:
@EikanWang , thanks for the catch. Fixed the logic.